tests/ami/utils.py

```diff
@@ -0,0 +1,23 @@
+from litestar import Litestar
+from litestar.testing import TestClient
+
+from app.auth import jwt_cookie_auth
+from app.models import User
+
+
+def login(user: User, test_client: TestClient[Litestar]) -> None:
+    response = jwt_cookie_auth.login(identifier=str(user.id))
+    test_client.cookies.update({jwt_cookie_auth.key: str(response.cookies[0].value)})
+
+
+async def check_url_without_auth(
```

Comment:
Nit: for clarity, could this helper be called something like `assert_query_fails_without_auth`?